nilearn/input_data/nifti_maps_masker.py

```diff
@@ -2,19 +2,30 @@
 Transformer for computing ROI signals.
 """
 
-from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.externals.joblib import Memory
 
 from .. import _utils
-from .._utils import logger
-from .._utils import CacheMixin
+from .._utils import logger, CacheMixin
+from .._utils.class_inspect import get_params
 from .._utils.niimg_conversions import _check_same_fov
 from .. import region
 from .. import image
-from .base_masker import filter_and_extract
+from .base_masker import filter_and_extract, BaseMasker
 
 
-class NiftiMapsMasker(BaseEstimator, TransformerMixin, CacheMixin):
+class _ExtractionFunctor(object):
```

Comment:
Sorry if I missed it, but the redundancy of the API with /nifti_labels_masker.py is wanted, isn't it ?